httpx/dispatch/http11.py

```diff
@@ -73,6 +77,12 @@ def __init__(
         """
         Send the request method, URL, and headers to the network.
         """
+        logger.debug(
+            f"send_request method={request.method!r} "
+            f"target={request.url.full_path!r} "
+            f"headers={request.headers!r}"
+        )
```

Comment:
We probably also want to log the sending of `h11.Data` and `h11.EndOfMessage` events, right?